StaggeredCoupledProcesses/TH/ThermalDensityDrivenFlow2D/thermal_gravity_driven_flow2d.prj

```diff
@@ -0,0 +1,304 @@
+        <process>
+            <name>LiquidFlow</name>
+            <type>LIQUID_FLOW</type>
+            <integration_order>2</integration_order>
+
+            <darcy_gravity>
+               <!-- axis_id: 0, 1, or the dimension of space minus one -->
+               <axis_id>1</axis_id>
+               <!-- g>=0. g=0: non gravity term -->
+               <g>9.81</g>
+            </darcy_gravity>
+
+            <process_variables>
+                <process_variable>pressure</process_variable>
+            </process_variables>
+
+            <secondary_variables>
+                <secondary_variable type="static" internal_name="darcy_velocity_x" output_name="v_x" />
+                <secondary_variable type="static" internal_name="darcy_velocity_y" output_name="v_y" />
+            </secondary_variables>
+
+            <material_property>
+                <fluid>
+                    <density>
+                       <type>TemperatureDependent</type>
+                       <temperature0> 293.0 </temperature0>
+                       <beta> 4.3e-4 </beta>
+                       <rho0>1000.</rho0>
+                    </density>
+                    <viscosity>
+                        <type>Constant</type>
+                        <value> 1.e-3 </value>
+                    </viscosity>
+                    <specific_heat_capacity>
+                       <type>Constant</type>
+                       <value> 4000. </value>
+                    </specific_heat_capacity>
+                    <thermal_conductivity>
+                       <type>Constant</type>
+                       <value> .6 </value>
+                    </thermal_conductivity>
+                </fluid>
+                <porous_medium>
+                    <porous_medium id="0">
+                        <permeability>
+                            <values>1.e-12</values>
+                        </permeability>
+                        <porosity>
+                            <type>Constant</type>
+                            <value> 0.2 </value>
+                        </porosity>
+                        <storage>
+                            <type>Constant</type>
+                            <value> 0.0 </value>
+                        </storage>
+                    </porous_medium>
+                </porous_medium>
+                <solid>
+                    <thermal_expansion>solid_thermal_expansion</thermal_expansion>
+                    <biot_constant>biot_const</biot_constant>
+                </solid>
+            </material_property>
+        </process>
+    </processes>
+
+    <time_loop>
+       <global_process_coupling>
+           <max_iter> 5 </max_iter>
+           <convergence_criterion>
+               <type>Residual</type>
+               <norm_type>NORM2</norm_type>
+               <reltol>1.e-4</reltol>
+           </convergence_criterion>
+       </global_process_coupling>
+        <processes>
+            <process ref="HeatConduction">
```

Comment:
Just follows the order of the processes defined in this project file. For instance, if heat process is defined first, then the coupling loop solves the heat process  equation firstly.